editor/src/clj/util/coll.clj

```diff
@@ -221,6 +221,7 @@
    (cond
      (empty? a) b
      (empty? b) a
+     (and (map? a) (supports-transient? a) (map? b)) (-> (reduce-kv assoc! (transient a) b) persistent! (with-meta (meta a)))
```

Comment:
A lot of map entries were created when compiling guis (`GuiSceneNode`'s `basic-gui-scene-info` output).